spec/portal/fixtures/download_certificate_failure.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Legally speaking, I think we're not allowed to just copy their HTML code, let's replace the whole file with just what's necessary, like a super simple string instead.